modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/hooks/useBannedDomains.js

```diff
@@ -14,30 +14,49 @@ import {useEffect, useState} from 'react';
 import {getBannedEmailDomains} from '../services/liferay/graphql/queries';
 import useDebounce from './useDebounce';
 
-export default function useBannedDomains(email) {
-	const debouncedEmail = useDebounce(email, 500);
-	const [bannedDomains, setBannedDomains] = useState(debouncedEmail);
+export default function useBannedDomains(value) {
+	const debouncedValue = useDebounce(value, 500);
```

Comment:
I think it will be better if declare a constant for this value (500) , what do you think?

Reply:
Done!